Framework/Algorithms/test/PhaseQuadMuonTest.h

```diff
@@ -17,6 +17,30 @@ using namespace Mantid::API;
 
 namespace {
 
+const int dead1 = 4;
+const int dead2 = 12;
+
+void populatePhaseTable2(ITableWorkspace_sptr phaseTable,
```

Comment:
What's the difference between this and `populatePhaseTable2`? Would something like `populatePhaseTableWithDeadDetectors` be more informative?

Reply:
ah, the phase table values are different